groups/script.js

```diff
@@ -72,9 +72,10 @@ const handler = {
         if (value) {
           renderGroupCreationModal({
             onClose: () => (dataStore.isGroupCreationModalOpen = false),
-            onSubmit: (inputValue) => {
+            onSubmit: ({ title, description }) => {
               return createDiscordGroupRole({
-                rolename: inputValue,
+                rolename: title,
+                ...(description && { description }),
```

Comment:
Ensure the validation logic is included when the form is submitted.  To implement the validation logic for the description field, ensuring it meets the requirements for length, special characters, and prohibited words.